.buildkite/pipeline-resource-definitions/kibana-deploy-project.yml

```diff
@@ -0,0 +1,44 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: bk-kibana-deploy-project
+  description: 'Builds and deploys a Kibana serverless project from a PR'
+  links:
+    - url: 'https://buildkite.com/elastic/kibana-deploy-project'
+      title: Pipeline link
+spec:
+  type: buildkite-pipeline
+  system: buildkite
+  owner: 'group:kibana-operations'
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: kibana / deploy project
```

Comment:
maybe some more descriptive naming could be useful here, highlighting that this is a PR-related deployment?

Reply:
I agree, something PR-related should be in the name. Maybe `kibana / deploy project from PR`?